library/src/scala/quoted/Expr.scala

```diff
@@ -18,6 +18,14 @@ class Expr[+T] private[scala] {
    */
   final def getValue[U >: T](given qctx: QuoteContext, valueOf: ValueOfExpr[U]): Option[U] = valueOf(this)
```

Comment:
I find `getValue` is a little misleading, if what it does is to get the literal out. Maybe `getLiteralValue`?

Reply:
It is not necessarily a literal.